### ISSUE TITLE
A11y_.NET Websites_Benchmark _ Collections _Screenreader: Narrator is announcing unnecessary information “Group” before we move to the editor. 

### ISSUE THREAD

**Issue by khansahab15** [dotnet/try#851]
**Environment Details:**
Application Name: .NET Websites
#URL:https://dotnet.microsoft.com/
Browser Versions: Microsoft Edge 44.18362.449.0
Windows Version: Windows10
Chrome: Version 79.0.3945.130 (Official Build) (64-bit)
 

**Example Page URL:** https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1

**Repro Steps:** 
1. Launch DotNet website application using URL: https://dotnet.microsoft.com/
2. Click on Learn link from the top navigation.
3. Click on Browser Tutorial link from the "Get started section".
4. "Step 6: Collections" screen gets open 
5. open narrator and move the focus to the editor using tab key 

**Actual:**
Narrator is announcing unnecessary information “Group” before we move to the editor.
Same issue with NVDA.

**Expected:**
Narrator should not announce unnecessary information “Group” before we move to the editor.
Narrator should announce control type for the editor as “Editor”.

**User Impact:**
Screen reader user may get confused and assume that the name of the editor is “Group”.

**Attachment:**
[#1721.pptx](https://github.com/dotnet/website/files/4138045/1721.pptx)

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

#A11yTCS; #.NET Websites; #A11y_.NET Websites_Benchmark; #DesktopWeb; #Win10; #Edge; #WCAG1.3.1; #Narrator; #NVDA; #Chrome

**Comment by terrimorton** [comment]
@LadyNaggaga Hi Maria, do you have an ETA on remediation? This bug (as well as #806) are blocking the accessibility compliance for dotnet.microsoft.com

**Comment by terrimorton** [comment]
Walked through these with GUy Barker on 4/23/2021.  These 2 no longer repro:

- Expected1: Narrator should not announce unnecessary information “Group” before we move to the editor.
- Expected2:Narrator should announce control type for the editor as “Editor”.

This one still repros:

- Expected3: Narrator should not announce Table using Caps+Arrow keys

@SureshNMS We should give the "code-actions" div a role of "presentation"
